README.md

```diff
@@ -19,13 +19,57 @@ Common requests are:
 
 The success of Sensu depends on its contributors. You can always help the maintainers by:
 
-* [Reviewing open PRs][4] (especially if you're familiar with problem it solves)
-* Helping review and comment on [open Issues][5] (especially older ones)
+* [Reviewing open PRs][4] (especially if you're familiar with technology involved)
+* Reviewing and commenting on [open Issues][5] (especially older ones)
 * Opening PRs that resolve [open Issues][5]
-* Volunteer as a maintainer (but only after you've done :point_up: a bunch first)
+* Volunteer as a maintainer (after you've done these :point_up: for some time)
 
 And **thank you** for your contribution. It adds to the monitoring :heart:.
 
+## Maintained Areas
+
+Maintainers do their best to cover the diverse set of plugins and adjacent technologies that make using Sensu a joy.
+
+| Area                         | Major Links                                                  | Maintainers                                                                                                                                        | Needs Contributors?                                                                              |
+|:-----------------------------|:-------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------------------------------------------------------------------------------------|
+| ![Sensu Org][sensu_pic]      | [Plugins Org][plug_org]</br>[Extensions Org][ext_org]        | **[Matt (@mbbroberg - Lead)][11]**</br>[Ben (@majormoses)][9]</br>[Eric (@eheydrick)][10]</br>[Luis (@luisdavim)][12]</br>[Shane (@sstarcher)][13] | **Yes!** [Read about how!][volunteer]                                                            |
+| ![Ruby Plugin][ruby_pic]     | [Ruby Plugin Library][ruby_lib]                              | **[Ben (@majormoses - Lead)][9]**</br>[Eric (@eheydrick)][10]</br>[Cameron (@cwjohnston)][17]                                                      | **Yes!** Please [claim an open issue!](https://github.com/sensu-plugins/sensu-plugin/issues)     |
+| ![Python Plugin][python_pic] | [Python Plugin Library][py_lib]                              | **[Barry O'Rourke (@barryorourke - Lead)][14]**                                                                                                    | **Yes!** [Help wanted on testing 🙏][py_helpwanted]                                              |
```

Comment:
If you want to add me here I started working some python for another project recently I can provide reviews and guidance even if I do not have the time to fully maintain it.